### ISSUE TITLE
Hi,

### ISSUE THREAD

**Issue by bekatoros** [mcguffin/woocommerce-payforpayment#30]

**Comment by bekatoros** [comment]
Hi,
I wanted to ask if you continue to support this plugin? And also if you need help with forum support?

**Comment by mcguffin** [comment]
Hi Marios,

I'm really thankful for your offer! Yes the plugin forum desperately needs a helping hand.
To put this a little further: I'd be the happiest guy if somebody else would offer to adopt the entire project, and keep it free and OpenSource. 
Initially I wrote the plugin for a client project, which I hoped would be a good starting point for more woocommerce setup jobs. Main reason I developed my own was that I was looking for a free solution, with regular updates and no subscription fees for my potential customers.
Long story short, It never got that far, and the shop is offline now (not my fault). So, aside from fame and glory I have no more good reasons to actively continue development. 
On the other hand it would be a pity to just abandon it, with its 100% 5star rating and a bunch of people counting on it.

I had a look at your portfolio, and the emspace.gr website.
Could it be that you are the man?

Best regards,
jörn

**Comment by bekatoros** [comment]
Hi Jörn,
Yes I can help with the support forum. I also can help with troubleshooting but I am not sure I will be able to debug and extend the plugin alone. 
You have done a very good job with the plugin, it works good ,it's stable and simple. It just needs from time to time updating the Wordpress tested up to values, in order to seem up to date.

Marios

**Comment by bekatoros** [comment]
Hi Jörn,
I fixed two errors that occurred one with Amazon payment gateway and a second with a shipping methods options.
I think that now the hooks are working better.
I have updated the plugin to 1.3.7, if you agree add the update it to the Wordpress repository.

Marios

**Comment by mcguffin** [comment]
Hi Marios,
thank you! I just commited everything to the WP repo. I reverted the version back to 1.3.6 (last version at WP was 1.3.5). 
cheers, jörn

**Comment by bekatoros** [comment]
Hi Jörn, 
can you push again the plugin to Wordpress repository.
I fixed an issue with chosen_methods and also after your last push the version in Wordpress is still 1.3.5.

Marios

**Comment by mcguffin** [comment]
it's done. For some reason the stable tag was still 1.3.5 in the 1.3.6 tag... Looks good now.
Thanks! jörn

**Comment by bekatoros** [comment]
Hi jörn,

can you reupload the latest version of pay 4 payment. If you see at the
download zip
https://downloads.wordpress.org/plugin/woocommerce-pay-for-payment.1.3.6.zip
the latest fixes are saved inside a trunk folder that appeared out of
nowhere.
It would be good if you change the version to 1.3.7 to issue an update.

Thanks,
Marios



2015-09-01 17:50 GMT+03:00 Jörn Lund <notifications@github.com>:

> it's done. For some reason the stable tag was still 1.3.5 in the 1.3.6
> tag... Looks good now.
> Thanks! jörn
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mcguffin/woocommerce-payforpayment/issues/30#issuecomment-136749399>
> .
>



-- 
Marios Bekatoros
